packages/TelegramClient-UI.package/TCUMessage.class/instance/addSender.st

```diff
@@ -0,0 +1,8 @@
+drawing
+addSender
+	
+	self addMorphFront: (self defaultTextMorph
+		contents: (self sender fullName);
```

Comment:
Are these defensive parenthesis @jb3rndt ?

Reply:
line 5 seems to work without parenthesis, the outermost are necessary

Reply:
How can I have missed those 🤔? Acually, I just spotted more of them in `TCCUser>>hash`.